src/test/java/uk/gov/gchq/magmacore/demo/ExampleSignsTest.java

```diff
@@ -12,21 +12,29 @@
  * the License.
  */
 
-package uk.gov.gchq.magmacore;
+package  uk.gov.gchq.magmacore.demo;
 
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNotNull;
 
 import org.junit.Test;
 
+import uk.gov.gchq.magmacore.service.MagmaCoreServiceFactory;
+
 /**
- * Unit test for simple App.
+ * Exercise the {@link ExampleSigns} code during the build.
  */
-public class AppTest {
+public class ExampleSignsTest {
+
     /**
-     * Rigorous Test.
-     */
+     * Test the {@link ExampleSigns} code.
+     *
+     * */
     @Test
-    public void shouldAnswerWithTrue() {
-        assertTrue(true);
+    public void testSignsExample() {
+        final var service = MagmaCoreServiceFactory.createWithJenaDatabase();
+        
+        final var transformation = ExampleSigns.populateExampleData(service);
+
+        assertNotNull(transformation);
```

Comment:
I believe it is even though it isn't a full unit test - it exercises code that would otherwise not be exercised during the build, which means that exceptions due to code changes won't be missed. I think of it as a sort of regression test. Obviously a full set of unit tests would be ideal, but since there are currently none there at all, this is definitely an improvement.

Reply:
Could we assert that `transformation` is populated with the expected data rather than just that it's not null?

Reply:
Not easily as there is quite lot of data and it's different each time because it uses random IRIs. I'd prefer to see a separate activity to create a set of proper unit tests for the whole code base - this test is good for exercising the example at build-time rather than expecting someone to run it by hand and forgetting, like with the other examples that are currently failing, clearly they would have been caught sooner if a test like this already existed.